tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -304,7 +304,7 @@ class CompletionArgSuite extends BaseCompletionSuite {
     s"""|case class A(argument: Int)
         |object Main {
         |  def foo(argument: Int): A =
-        |    A(arg@@)
+        |    A(argu@@)
```

Comment:
I had to change these cases.

After [this fix](https://github.com/lampepfl/dotty/commit/f47c8c64af8382d1d41261d8b53fb18624243175) in dotty, it started returning package names in completions.
In this suite we have cases that enters packages with `arg` prefix ("arg1", "arg2") into compiler state and for latest version completions returns them too.

Unfortunately, I haven't found a way to cleanup them. Added some info about this problem in [this comment](https://github.com/scalameta/metals/issues/917#issuecomment-1080835816)